### PR TITLE
Move machine defaults + Hide kubevirt platform in survey

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: pkg/asset/machines/worker.go [hunk 1]
```diff
@@ -146,6 +146,14 @@ func defaultVSphereMachinePoolPlatform() vspheretypes.MachinePool {
 	}
 }
 
+func defaultKubevirtMachinePoolPlatform() kubevirttypes.MachinePool {
```
Comment (ravidbro) [inline pkg/asset/machines/worker.go]: Why is the default node size need to be per platform?
I would expect that OpenShift nodes of the tenant cluster run the same software and will have the same requirement.

Comment (nirarg) [reply pkg/asset/machines/worker.go]: The size parameters can be different per platform, for example GCP has the `InstanceType` parameter
Therefore there should be specific function for each platform

Comment (ravidbro) [reply pkg/asset/machines/worker.go]: still...

Comment (ravidbro) [reply pkg/asset/machines/worker.go]: If all the platforms do the same then I guess it's OK

**Review by bardielle — APPROVED** [review]

**Pull request merged**